### PR TITLE
Add Vincent profile and publications

### PR DESCRIPTION
**Pull request by vdutor**

### CODE REVIEW

File: _people/vincent-dutordoir.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+crsid: vd309
+email: vd309@cam.ac.uk
+supervisor: che29
+position: PhD Student
+institution: Cambridge University
+image: vincentd.jpg
+biography: >-
+	Vincent is a PhD student in the Computational and Biological Learning Lab (CBL)
+	with Prof. Zoubin Ghahramani, and in the Machine Learning at Computer Lab
+	(ML@CL) with Prof. Carl Henrik Ek. He is also a member of Jesus College,
```
Comment (apaleyes) [inline _people/vincent-dutordoir.md]: Carl is great, but not a "Prof" yet

**Review by apaleyes — COMMENTED** [review]
Looks good, just one correction below

File: _people/vincent-dutordoir.md (hunk 1 above)
Comment (vdutor) [inline _people/vincent-dutordoir.md]:
```suggestion
	with Zoubin Ghahramani, and in the Machine Learning at Computer Lab
	(ML@CL) with  Carl Henrik Ek. He is also a member of Jesus College,
```

File: _people/vincent-dutordoir.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	Computer Science Engineering from Ghent University (Belgium) in 2017. Broadly
+	speaking, his interests are in models for uncertainty-aware decision making,
+	with a special focus on (deep) Gaussian processes and Neural Diffusion
+	Processes.
+publications:
+  - sparse-gaussian-processes-with-spherical-harmonic-features
+  - deep-neural-networks-as-point-estimates-for-deep-gaussian-processes
+---
+
+Vincent is a PhD student in the Computational and Biological Learning Lab (CBL) with Prof. Zoubin Ghahramani, and in the Machine Learning at Computer Lab (ML@CL) with Prof. Carl Henrik Ek. He is also a member of Jesus College, University of Cambridge.
```
Comment (vdutor) [inline _people/vincent-dutordoir.md]:
```suggestion
Vincent is a PhD student in the Computational and Biological Learning Lab (CBL) with Zoubin Ghahramani, and in the Machine Learning at Computer Lab (ML@CL) with Carl Henrik Ek. He is also a member of Jesus College, University of Cambridge.
```

File: _people/vincent-dutordoir.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+crsid: vd309
+email: vd309@cam.ac.uk
+supervisor: che29
+position: PhD Student
+institution: Cambridge University
+image: vincentd.jpg
+biography: >-
+	Vincent is a PhD student in the Computational and Biological Learning Lab (CBL)
+	with Zoubin Ghahramani, and in the Machine Learning at Computer Lab
+	(ML@CL) with  Carl Henrik Ek. He is also a member of Jesus College,
```
Comment (vdutor) [inline _people/vincent-dutordoir.md]:
```suggestion
	(ML@CL) with Carl Henrik Ek. He is also a member of Jesus College,
```

**Pull request merged**